### PR TITLE
added AsishRaju

### PR DESCRIPTION
**Pull request by AsishRaju**
## Proposed Changes

-  Added my pic in `src\photos\AsishRaju.jpg`
-  Added my details in `src\components\json\worldData.js`

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -119,6 +120,21 @@ const data = [
...
 		twitter: "https://twitter.com/iamKarthikBK",
 		portfolio: "https://www.nohello.com",
+	},
+	{
+		longitude: 83.376702,
+		latitude: 17.781218,
+		batch: 2020,
+		photo: AsishRaju,
+		region: "Asia",
+		skills: ["Full Stack Web", "UI/UX",],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:130]:
```suggestion
		skills: ["Web Dev", "UI/UX",],
```

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -119,6 +120,21 @@ const data = [
...
+		name: "V Asish Raju",
+		bio: "I would love to change the world, but they won’t give me the source code",
+		url: "https://dsc.community.dev/gandhi-institute-of-technology-and-management-visakhapatnam-campus/",
+		chapterName: "GITAM University, Vizag Campus",
+		github: "https://github.com/AsishRaju",
+		linkedin: "https://www.linkedin.com/in/asishraju/",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:136]:
```suggestion
		linkedin: "https://linkedin.com/in/asishraju",
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@AsishRaju You have used an invalid skill type.

**Comment by AsishRaju** [comment]
@aravindvnair99 changes done 👍🏻

**Review by abhinavsri360 — APPROVED** [review]
Great work!

**Pull request merged**